frontend/src/views/Game.tsx

```diff
@@ -1,31 +1,68 @@
 import Editor from '@monaco-editor/react';
 import React, { useEffect, useState } from 'react';
 import { useLocation } from 'react-router-dom';
+import styled from 'styled-components';
+import { Problem, getProblems } from '../api/Problem';
 import { Room } from '../api/Room';
+import { Text, ProblemHeaderText } from '../components/core/Text';
+import Header from '../components/navigation/Header';
 
 type LocationState = {
   room: Room,
 }
 
-function GamePage() {
+const FlexContainer = styled.div`
+  display: flex;
+  top: 50px;
+`;
+
+const FlexPanel = styled.div`
+  flex: 1;
+  padding: 1rem;
+  background-color: #e3e3e3;
+  overflow: none;
+  height: 100vh;
+  width: 50vw;
```

Comment:
I'm having trouble understanding if some of these attributes are actually contributing anything? Notably, the following don't seem to be doing anything:
```
flex: 1;
width 50vw;
```
Since you already have the `flex: 1` value, I don't know that the others are necessary?

I'd also consider looking into how you can update this to make the height of "Header + Problem & Code Panel" equal to the screen height (I also referenced this in a comment below).

Reply:
Ok I got rid of those two lines, they may have been from an old commit that got mixed in